enforcer-rules/src/test/java/com/google/cloud/tools/dependencies/enforcer/LinkageCheckerRuleTest.java

```diff
@@ -75,18 +75,21 @@
   private DependencyResolutionResult mockDependencyResolutionResult;
 
   @Before
-  public void setup() {
+  public void setup()
+      throws ExpressionEvaluationException, ComponentLookupException,
+          DependencyResolutionException {
     rule = new LinkageCheckerRule();
     repositorySystem = RepositoryUtility.newRepositorySystem();
     repositorySystemSession = RepositoryUtility.newSession(repositorySystem);
+    setupMock();
```

Comment:
With line 92, `repositorySystemSession` needs to be instantiated before that. Prior to this PR, there's no order specified between `setup` and `setupMock()`